src/journal-remote/journal-upload.c

```diff
@@ -35,7 +36,7 @@
 
 #define PRIV_KEY_FILE CERTIFICATE_ROOT "/private/journal-upload.pem"
 #define CERT_FILE     CERTIFICATE_ROOT "/certs/journal-upload.pem"
-#define TRUST_FILE    CERTIFICATE_ROOT "/ca/trusted.pem"
+#define TRUST_FILE    CERTIFICATE_ROOT "/certs/ca-certificates.crt"
```

Comment:
Please make that a separate commit, with rationale. (I honestly don't know whether we should make this change. The reasoning needs to be recorded in the commit message.)

Reply:
Agreed.